apps/explorer/src/hooks/useGetTransactionBlocks.ts

```diff
@@ -32,7 +32,10 @@ export function useGetTransactionBlocksForAddress(
         {
             getNextPageParam: (lastPage) =>
                 lastPage?.hasNextPage ? lastPage.nextCursor : false,
-            enabled: !!address,
+            staleTime: Infinity,
```

Comment:
We probably want to change this to like 10 seconds honestly, not 100% sure why this got bumped to infinite